flopy/mf6/utils/codegen/templates/package_container.jinja

```diff
@@ -0,0 +1,64 @@
+class {{ name.title.title() }}Packages(mfpackage.MFChildPackages):
+    """
+    {{ name.title.title() }}Packages is a container class for the Modflow{{ name.title.title() }} class.
+
+    Methods
+    -------
+    initialize
+        Initializes a new Modflow{{ name.title.title() }} package removing any sibling child
+        packages attached to the same parent package. See Modflow{{ name.title.title() }} init
+        documentation for definition of parameters.
+    append_package
+        Adds a new Modflow{{ name.title.title() }} package to the container. See Modflow{{ name.title.title() }}
+        init documentation for definition of parameters.
+    """
+
+    package_abbr = "{{ name.title.lower() }}packages"
+
+    def initialize(
+        self,
+        {%- for n, var in variables.items() if var.container_init_param and not var.init_skip %}
+        {%- if var.default is defined %}
+        {{ n }}={{ var.default }},
+        {%- else -%}
+        {{ n }},
+        {% endif -%}
```

Comment:
In theory this could be a one-liner, but I don't know if it's more readable.

(untested)
```suggestion
        {{ n }}{%- if var.default %}={{ var.default }}{% endif -%},
```

Reply:
changed as suggested